src/fsharp/FSharp.DependencyManager/FSharp.DependencyManager.Utilities.fs

```diff
@@ -0,0 +1,272 @@
+            let valueOpt =
+                let valueText =
+                    if pos < 0 then option
+                    else if pos < option.Length then
+                        option.Substring(pos + 1)
+                    else ""
+                stringAsOpt (valueText.Trim(trimChars))
+            nameOpt,valueOpt
+
+        let last = String.length text - 1
+        let result = ResizeArray()
+        let mutable insideSQ = false
+        let mutable start = 0
+        let isSeperator c = c = ','
+        for i = 0 to last do
+            match text.[i], insideSQ with
+            | c, false when isSeperator c ->                        // split when seeing a separator
+                result.Add(text.Substring(start, i - start))
+                insideSQ <- false
+                start <- i + 1
+            | _, _ when i = last ->
+                result.Add(text.Substring(start, i - start + 1))
+            | c, true when isSeperator c ->                         // keep reading if a separator is inside quotation
+                insideSQ <- true
+            | '\'', _ when isNotQuotedQuotation text i ->           // open or close quotation
+                insideSQ <- not insideSQ                            // keep reading
+            | _ -> ()
+
+        result |> Seq.map(fun option -> split option)
+
+    let fsharpCompilerPath = Path.GetDirectoryName(typeof<DependencyManagerAttribute>.GetTypeInfo().Assembly.Location)
+    let pathToLoggerDll = Path.Combine(fsharpCompilerPath, @"FSharp.Build.dll")
+    let pathToDotNetExe = Path.Combine(fsharpCompilerPath, @"..\..\..\dotnet.exe")
+
+    // TODO:    Do a better job of this
+    let msbuild projectPath arguments =
+        // Need to do a much better job of this
+        let psi = ProcessStartInfo()
+        psi.FileName <- pathToDotNetExe
+        psi.RedirectStandardOutput <- false
+        psi.RedirectStandardError <- false
+        psi.Arguments <- arguments
+        psi.CreateNoWindow <- true
+        psi.UseShellExecute <- false
+        use p = new Process()
+        p.StartInfo <- psi
+        p.Start() |> ignore
+        p.WaitForExit()
+        let exitCode = p.ExitCode
+        exitCode = 0
+
+    let buildProject projectPath verbosity =
+        let outputFile = projectPath + ".fsx"
+        let logFile = projectPath + ".log"
+
+#if FX_RESHAPED_MSBUILD
+        // Miserably we can't use the in-process msbuild on coreclr --- we would essentially have to re-ship the whole bloody thing.
+        let verbosityToArgument =
+            match verbosity with
+            | "diag" -> @"/v:diag"
+            | "detailed" -> @"/v:detailed"
+            | "normal" -> @"/v:normal"
+            | "quiet" ->  @"/v:quiet"
+            | _ ->  @"/v:minimal"
+
+        let arguments = sprintf "msbuild  /t:FSI-PackageManagement %s /logger:%s;%s;%s %s" verbosityToArgument pathToLoggerDll logFile verbosity projectPath
```

Comment:
`dotnet msbuild` doesnt know the path to .NET gac on mono.

it's possibile to use `msbuild` on mono.

or set `FrameworkPathOverride` property the (like https://github.com/enricosada/dotnet-proj-info/blob/master/.travis.yml#L8 ) to tell the .net sdk where the gac is